sdk/src/asset_handlers/sfnt_io.rs

```diff
@@ -322,7 +322,9 @@ impl SfntFont {
                     size_of::<SfntDirectoryEntry>() as i64
                 } else {
                     // We added some other number of tables
-                    return Err(wrap_font_err(FontError::SaveError));
+                    return Err(FontError::SaveError(
+                        FontSaveError::UnexpectedNumberOfTables,
```

Comment:
I think here instead maybe we have a `TooManyTablesAdded` error to be more specific